src/hammer-vlsi/hammer_vlsi/hammer_tool.py

```diff
@@ -889,6 +889,17 @@ def get_all_ground_nets(self) -> List[Supply]:
     def get_independent_ground_nets(self) -> List[Supply]:
         return list(filter(lambda x: x.tie is None, self.get_all_ground_nets()))
 
+    def get_bumps(self) -> Optional[BumpsDefinition]:
+        bumps = self.get_setting("vlsi.inputs.bumps")
+        if bumps is None or bumps == "null":
```

Comment:
Is the string `"null"` possible? I thought we made it so that it is converted to `None`.

Reply:
Yeah you are right I can remove that string check.